org.lflang/src/org/lflang/generator/c/CCmakeGenerator.java

```diff
@@ -247,4 +250,36 @@ CodeBuilder generateCMakeCode(
         
         return cMakeCode;
     }
+    
+    /**
+     * Add the linked libraries in case any were specified.
+     * @param cb The code builder to append to.
+     */
+    private void addLinkedLibraries(CodeBuilder cb) {
+        StringBuffer block = new StringBuffer();
+        for (var lib : targetConfig.linkLibraries) {
+           var findLib = "find_library( LF_%1$s_LIB %1$s )".formatted(lib);
+           var linkLibs = """
+                   target_link_libraries( ${LF_MAIN_TARGET} "${LF_%s_LIB}")""".formatted(lib);
+           switch (lib.trim()) {
+                case "m":
```

Comment:
If we are only special casing the math library, this function might be an overkill. 
If we plan to special case other libraries, I think going with`cmake-include` is a better solution (as mentioned [in #1120](https://github.com/lf-lang/lingua-franca/issues/1120#issuecomment-1113009360)).